### PR TITLE
move update guide to learn>advanced topics

### PR DESCRIPTION
**Pull request by maryamsulemani97**

### CODE REVIEW

File: .vuepress/public/_redirects [hunk 1]
```diff
@@ -79,3 +79,6 @@
 /reference/api/attributes_for_faceting.html         /reference/api/filterable_attributes.html
 /reference/features/filtering.html                   /reference/features/filtering_and_faceted_search.html
 /reference/features/faceted_search.html              /reference/features/filtering_and_faceted_search.html
+
+#move updating MeiliSearch
+/create/how_to/updating.md)                          /learn/advanced/updating.md
```
Comment (guimachiavelli) [inline .vuepress/public/_redirects:84]:
```suggestion
/create/how_to/updating.md                           /learn/advanced/updating.md

```

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
Just needs a typo fix and a new line at the end and we're good to go 👽

File: .vuepress/config.js [hunk 2]
```diff
@@ -81,6 +81,7 @@ module.exports = {
           children: [
             '/learn/advanced/asynchronous_updates',
             '/learn/advanced/snapshots_vs_dumps',
+            '/learn/advanced/updating.md',
```
Comment (dichotommy) [inline .vuepress/config.js:84]: Not 100% certain but pretty sure that if we don't specify a title, VuePress will use the first H1 in the file, which in this case is `# Update to the latest MeiliSearch version`. Seems a bit long to me, therefore I prefer:

```suggestion
            {
              title: 'Updating MeiliSearch',
              path: '/learn/advanced/updating.md'
            },
```

File: .vuepress/public/_redirects [hunk 3]
```diff
@@ -79,3 +79,6 @@
 /reference/api/attributes_for_faceting.html         /reference/api/filterable_attributes.html
 /reference/features/filtering.html                   /reference/features/filtering_and_faceted_search.html
 /reference/features/faceted_search.html              /reference/features/filtering_and_faceted_search.html
+
+#move updating MeiliSearch
+/create/how_to/updating.md                           /learn/advanced/updating.md
```
Comment (dichotommy) [inline .vuepress/public/_redirects:84]: This redirection occurs through Netlify, so `.md` links will not work.

```suggestion
/create/how_to/updating.html                           /learn/advanced/updating.html
```

File: index.md [hunk 4]
```diff
@@ -15,14 +15,15 @@ Welcome to the MeiliSearch documentation. Here you'll find everything you need t
 **Other helpful links**:
 
 - [Install MeiliSearch](/learn/getting_started/installation.md)
+- [Update MeiliSearch](/learn/advanced/updating.md)
```
Comment (dichotommy) [inline index.md:18]:
```suggestion
- [Updating MeiliSearch](/learn/advanced/updating.md)
```

**Review by dichotommy — CHANGES_REQUESTED** [review]
Thanks for doing this, just a few changes 🙃

**Comment by dichotommy** [comment]
Oops, because of my suggestion you're missing a comma on line 86 of `config.js`.

**Review by guimachiavelli — APPROVED** [review]
LGTM

**Review by dichotommy — APPROVED** [review]
LGTM 🤠👍🏻

**Comment by dichotommy** [comment]
bors merge

**Pull request merged**